app/questionnaire/questionnaire_schema.py

```diff
@@ -893,3 +893,10 @@ def _get_rules_section_dependencies(
                 )
 
         return rules_section_dependencies
+
+    def _get_related_answers_for_section(self, section_id: str) -> Optional[list[str]]:
```

Comment:
Can avoid type hinting by doing something like this?
```
          if summary := self.get_section(section_id).get("summary"):
          if related_answers := summary.get("related_answers"):
              return [
                  related_answer.get("identifier") for related_answer in related_answers
              ]
```

Reply:
I removed one type ignore but the other one was impossible to remove so I added a comment explaining why it needs to stay, thanks!